src/test/java/util/SetupTest.java

```diff
@@ -0,0 +1,42 @@
+package util;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.openqa.selenium.Capabilities;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import pageclasses.WebDriverSingleton;
+
+@ExtendWith(FailedTestWatcher.class)
+public class SetupTest {
```

Comment:
if all test classes are inherited from this class, then it's better to name it BaseTest